manifest/manifest_ontology.ttl

```diff
@@ -0,0 +1,26 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix gax-core: <https://w3id.org/gaia-x/core#> .
+@prefix manifest: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/manifest/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+
+manifest: a owl:Ontology ;
+    rdfs:label "ontology definition for Manifest"@en ;
+    dcterms:contributor "Johannes Demer (ASCS)" ;
+    owl:versionInfo "0.1" .
+
+manifest:Manifest a owl:Class ;
+    rdfs:label "class definition for Manifest" ;
+    rdfs:comment "Manifest attributes for all assets"@en ;
+    rdfs:subClassOf gx:DataResource .
```

Comment:
is it really a gx:DataResource or should it be something what you want to link into another Ontology like georeference?

Reply:
it needs to be on the same level like, the Domain metadata like hdmap, ositrace, ...